### PR TITLE
Improve build experience

### PR DESCRIPTION
**Pull request by AlekseyMartynov**
* First commit - simple cleanup of project.json files
* Second commit - to solve issues described in https://github.com/DevExpress/DevExtreme.AspNet.TagHelpers/issues/36

@San4es please review

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+dotnet restore
+
+pushd DevExtreme.AspNet.TagHelpers.Generator
+dotnet run || (
+    echo "Error: Generator failed, please report this bug"
+    exit /b 1
+)
+popd
+
+dotnet build DevExtreme.AspNet.TagHelpers
```
Comment (San4es) [inline build.cmd]: @AlekseyMartynov 

Where is a new line at the end of this file? :)

File: Samples/project.json [hunk 2]
```diff
@@ -64,6 +64,10 @@
                 "node_modules"
             ]
         }
+    },
+
+    "scripts": {
+        "precompile": "../build.cmd"
     }
 
 }
```
Comment (San4es) [inline Samples/project.json]: @AlekseyMartynov 

Looks bad for me ;)

Comment (AlekseyMartynov) [inline Samples/project.json]: Are you on Mac ?

File: build.cmd (hunk 1 above)
Comment (AlekseyMartynov) [inline build.cmd]: Added. Thanks.

File: Samples/project.json (hunk 2 above)
Comment (San4es) [inline Samples/project.json]: No, I meant absent new line at the end of file :)

Comment (San4es) [inline Samples/project.json]: I think we can do this in separate PR to avoid unnecessary differences.
